Makefile

```diff
@@ -1,3 +1,8 @@
 update_metadata:
 	(cd libPhoneNumberTests && ./metadataGenerator) && \
-		(cd libPhoneNumber && ./GeneratePhoneNumberHeader.sh)
+		(cd libPhoneNumber && ./GeneratePhoneNumberHeader.sh) && \
+		(cp libPhoneNumber/NBGeneratedPhoneNumberMetaData.h MetaDataPlistCreator) && \
+		(cd MetaDataPlistCreator && MetaDataPlistCreator) && \
```

Comment:
I'm definitely uncomfortable with checking the binary into the repository. I guess the alternatives would be "invoke xcodebuild" or "translate this to a Swift script so you can run it on demand". How do you feel about the latter?

Reply:
I can check tomorrow whether a swift script will work